pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -31,67 +26,6 @@ var bundleCheck = Check{
 	labels: None,
 }
 
-var genericPreflightChecks = []Check{
-	{
-		configKeySuffix:  "check-admin-helper-cached",
-		checkDescription: "Checking if crc-admin-helper executable is cached",
-		check:            checkAdminHelperExecutableCached,
-		fixDescription:   "Caching crc-admin-helper executable",
-		fix:              fixAdminHelperExecutableCached,
-
-		labels: None,
-	},
-	{
-		configKeySuffix:  "check-obsolete-admin-helper",
-		checkDescription: "Checking for obsolete admin-helper executable",
-		check:            checkOldAdminHelperExecutableCached,
-		fixDescription:   "Removing obsolete admin-helper executable",
-		fix:              fixOldAdminHelperExecutableCached,
-	},
-	{
-
-		configKeySuffix:  "check-supported-cpu-arch",
-		checkDescription: "Checking if running on a supported CPU architecture",
-		check:            checkSupportedCPUArch,
-		fixDescription:   "CodeReady Containers is only supported on x86_64 hardware",
-		flags:            NoFix,
-
-		labels: None,
-	},
-	{
-		configKeySuffix:  "check-ram",
-		checkDescription: "Checking minimum RAM requirements",
-		check: func() error {
-			return validation.ValidateEnoughMemory(constants.DefaultMemory)
-		},
-		fixDescription: fmt.Sprintf("crc requires at least %s to run", units.HumanSize(float64(constants.DefaultMemory*1024*1024))),
-		flags:          NoFix,
-
-		labels: None,
-	},
-	{
-		cleanupDescription: "Removing CRC Machine Instance directory",
-		cleanup:            removeCRCMachinesDir,
-		flags:              CleanUpOnly,
-
-		labels: None,
-	},
-	{
-		cleanupDescription: "Removing older logs",
-		cleanup:            removeOldLogs,
-		flags:              CleanUpOnly,
-
-		labels: None,
-	},
-	{
-		cleanupDescription: "Removing pull secret from the keyring",
-		cleanup:            cluster.ForgetPullSecret,
-		flags:              CleanUpOnly,
-
-		labels: None,
-	},
```

Comment:
Are these last 3 cleanups already done by the (un)installer? Or should we keep these a bit longer?

Reply:
No this is not handled by the installer, we should keep them, will update